content/en/apps/features/muting/_index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Muting
+weight: 10
```

Comment:
I would actually put this after UHC Mode, before App Management

Reply:
Done